tests/Scopes.php

```diff
@@ -33,4 +33,24 @@ public function testMultipleScopes()
         $query = $mapper->select()->noQuote()->free()->active();
         $this->assertEquals("SELECT * FROM test_events  WHERE (test_events.type = ?) AND (test_events.status = ?)", $query->toSql());
     }
+
+    public function testEntityScopes()
+    {
+        $mapper = test_spot_mapper('SpotTest\Entity\Post');
+        $query = $mapper->select()->noQuote()->active();
+        $this->assertEquals("SELECT * FROM test_posts  WHERE test_posts.status = ?", $query->toSql());
+    }
+
+    public function testRelationScopes()
+    {
+        $mapper = test_spot_mapper('SpotTest\Entity\Post');
+        $mapper->insert([
+            'title' => 'Test',
+            'body' => 'Test body',
+            'author_id' => 1,
+            ]);
+        $query = $mapper->get(1)->comments->yesterday()->query();
+        $sql = str_replace(['`', '"'], '', $query->toSql());
```

Comment:
I tried that, but there are still backticks in the WHERE and ORDER BY clauses when I use `noQuote`.